pkg/util/util.go

```diff
@@ -120,3 +130,99 @@ func ReadBackingImageConfigFile(workDirectory string) (*BackingImageConfig, erro
 	}
 	return cfg, nil
 }
+
+func Execute(envs []string, binary string, args ...string) (string, error) {
+	return ExecuteWithTimeout(time.Minute, envs, binary, args...)
+}
+
+func ExecuteWithTimeout(timeout time.Duration, envs []string, binary string, args ...string) (string, error) {
+	var err error
+	cmd := exec.Command(binary, args...)
```

Comment:
You can use a CommandContext, then pass a context.withTimeout. This way you don't need the timeout go routine below.

Reply:
Addressed.